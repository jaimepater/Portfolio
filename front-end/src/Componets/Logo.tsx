import * as React from 'react';


const Logo = () => (
  <svg xmlns="http://www.w3.org/2000/svg" width="30" height="44" viewBox="0 0 56 84">
    <text transform="translate(0 51)" fill="#fff" fontSize="60" fontFamily="RageItalic, Rage"><tspan x="0" y="0">J</tspan></text>
    <text transform="translate(25 8)" fill="#fff" fontSize="60" fontFamily="RageItalic, Rage"><tspan x="0" y="51">p</tspan></text>
  </svg>
);


export default Logo;
