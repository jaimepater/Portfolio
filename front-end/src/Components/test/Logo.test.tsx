import React from "react";
import { shallow } from 'enzyme';
import Logo from "../Logo";


describe('Logo', () => {
  it('Renders Logo', () => {
    const shallowComponent = shallow(<Logo  />);
    expect(shallowComponent).toMatchSnapshot();
  });
});