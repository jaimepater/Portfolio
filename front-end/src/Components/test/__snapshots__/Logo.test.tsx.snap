// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Renders Logo 1`] = `
<svg
  height="44"
  viewBox="0 0 56 84"
  width="30"
  xmlns="http://www.w3.org/2000/svg"
>
  <text
    fill="#fff"
    fontFamily="Italianno"
    fontSize="60"
    transform="translate(0 51)"
  >
    <tspan
      x="0"
      y="0"
    >
      J
    </tspan>
  </text>
  <text
    fill="#fff"
    fontFamily="Italianno"
    fontSize="60"
    transform="translate(25 8)"
  >
    <tspan
      x="0"
      y="51"
    >
      p
    </tspan>
  </text>
</svg>
`;
